boa/integrations/jupyter/constants.py

```diff
@@ -1,13 +1,14 @@
 from datetime import timedelta
 
+NUL = b"\0"
 CALLBACK_TOKEN_TIMEOUT = timedelta(minutes=3)
-MEMORY_LENGTH = 50 * 1024  # Size of the shared memory object
```

Comment:
hm -- was this a bug?

Reply:
This is just to improve the 1 byte difference in the error message

Reply:
if you sent a string with 51200 chars, the error would say "Request body has 51201 bytes, but only 51200 are allowed"
which was incorrect. Now I made instead the shared memory 51201 bytes long, so the actual payload may be 51200.